tasks/db_deploy/migrations/migrate_versions_4_to_5/migrate_sql.py

```diff
@@ -0,0 +1,311 @@
+    """
+    Full SQL for creating the reconcile_job table. 
+
+    Returns:
+        (sqlalchemy.sql.element.TextClause): SQL for creating reconcile_job table.
+    """
+    return text(
+        """
+        -- Create reconcile_job table
+        CREATE TABLE IF NOT EXISTS reconcile_job  
+        ( 
+          id                     	BIGSERIAL NOT NULL
+        , orca_archive_location  	text NOT NULL
+        , status_id              	int2 NOT NULL
+        , inventory_creation_time	timestamp with time zone NOT NULL
+        , start_time             	timestamp with time zone NOT NULL
+        , last_update            	timestamp with time zone NOT NULL
+        , end_time               	timestamp with time zone NULL
+        , error_message          	text NULL
+        , CONSTRAINT PK_reconcile_job PRIMARY KEY(id)
+        , CONSTRAINT FK_reconcile_job_status FOREIGN KEY(status_id) REFERENCES reconcile_status(id)
+        );
+
+        -- Comments
+        COMMENT ON TABLE reconcile_job IS 'Manages internal reconciliation job information.';
+        COMMENT ON COLUMN reconcile_job.id IS 'Job ID unique to each internal reconciliation job.';
+        COMMENT ON COLUMN reconcile_job.orca_archive_location IS 'ORCA S3 Glacier bucket the reconciliation targets.';
+        COMMENT ON COLUMN reconcile_job.status_id IS 'Current status of the job.';
+        COMMENT ON COLUMN reconcile_job.inventory_creation_time IS 'Inventory report creation time from the s3 manifest.';
+        COMMENT ON COLUMN reconcile_job.start_time IS 'Date and time the internal reconcile job started.';
+        COMMENT ON COLUMN reconcile_job.last_update IS 'Date and time the job status was last updated.';
+        COMMENT ON COLUMN reconcile_job.end_time IS 'Time the job completed and wrote the report information.';
+        COMMENT ON COLUMN reconcile_job.error_message IS 'Critical error the job ran into that prevented it from finishing.';
+        """
+    )
+
+
+def reconcile_s3_object_table_sql() -> TextClause:
+    """
+    Full SQL for creating the reconcile_s3_object table. 
+
+    Returns:
+        (sqlalchemy.sql.element.TextClause): SQL for creating reconcile_s3_object table.
+    """
+    return text(
+        """
+            -- Create reconcile_s3_object table
+            CREATE TABLE IF NOT EXISTS reconcile_s3_object  ( 
+              job_id               	int8 NOT NULL
+            , orca_archive_location	text NOT NULL
+            , key_path             	text NOT NULL
+            , etag                 	text NOT NULL
+            , last_update          	timestamp with time zone NOT NULL
+            , size_in_bytes        	int8 NOT NULL
+            , storage_class        	text NOT NULL
+            , delete_marker        	bool NOT NULL
+            , CONSTRAINT FK_reconcile_job_s3_objects FOREIGN KEY(job_id) REFERENCES reconcile_job(id)
+            , CONSTRAINT PK_reconcile_s3_object PRIMARY KEY(orca_archive_location, key_path)
```

Comment:
Depending on what comes out of my PR, delete_marker and the keys may change.

Reply:
https://github.com/nasa/cumulus-orca/pull/241#discussion_r805021450

Reply:
Updated table name definition and moved constraints to be created at the partition level in order to handle the older PostgreSQL engine (10.14) that PG Aurora Serverless uses.